src/layouts/ContestLayout.js

```diff
@@ -94,14 +98,24 @@ const ContestLayout = (props) => {
                 </a>
               ) : null}
 
-              {t.contestStatus === "active" && findingsRepo && fields.submissionPath ? (
+              {t.contestStatus === "active" &&
+              findingsRepo &&
+              fields.submissionPath ? (
                 <Link
                   to={fields.submissionPath}
                   className="button cta-button button-medium secondary"
                 >
                   Submit Finding
                 </Link>
               ) : null}
+              {canViewReport ? (
+                <Link
+                  to={reportUrl}
+                  className="button cta-button button-medium secondary"
+                >
+                  View Report
```

Comment:
Did you know that instead of:
``` jsx
{condition ? <component/> : null}
```
you can do:
``` jsx
{condition && <component/>}
```
Just FYI. No need to change here. We use the former syntax in plenty of places. Plus, you could argue that the former is more readable. Maybe we can decide as a team which syntax we prefer.

Reply:
I DID NOT, that's super cool!